http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -609,6 +610,12 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       // delete the namespace only after the helm uninstall completes
       _ <- kubeService.deleteNamespace(dbApp.cluster.getGkeClusterId,
                                        KubernetesNamespace(dbApp.app.appResources.namespace.name))
+
+      fa = kubeService
```

Comment:
Thinking we may not need this if the test is the problem (see my comment above). It probably doesn't hurt though.

Reply:
I think this will still help...at very least, an error will be persisted to DB when namespace fail to delete